pkg/sql/schemachanger/scbuild/internal/scbuildstmt/drop_index.go

```diff
@@ -25,26 +25,13 @@ import (
 	"github.com/cockroachdb/errors"
 )
 
-// TODO (xiang): To enable full support for DROP INDEX, we still need to
+// TODO (xiang): To remove fallbacks for DROP INDEX we still need to:
 //   1. Model adding and dropping FK constraints correctly
 //      for dropping an index with dependent FK constraint;
-//   2. Fix the "revertibility bug" in the planner as it currently
-//      seems to ignore non-revertible operations and plan
-//      them to happen in a much early phase (say, stmt phase);
 //   2.a. Once the above two are done, add a test where
 //        we drop an index with a dependent FK constraint and a
 //        dependent view to end-to-end test, scbuild test, scplan test.
-//   2.b. Answer the question "Are there any CCL required `DROP INDEX` usage?"
-//        If yes, add a end-to-end test in `pkg/ccl/schemachangerccl`.
 //   3. Check if requires CCL binary for eventual zone config removal.
```

Comment:
Zone configs are not a CCL feature, though. I'm curious what this is about. @Xiang-Gu do you remember, by any chance? I realize that it's been a while!